build/cmake/lib/base/CMakeLists.txt

```diff
@@ -26,28 +24,24 @@ endif()
 
 wx_add_library(wxbase IS_BASE ${BASE_FILES})
 
-if(NOT wxBUILD_MONOLITHIC)
-    wx_lib_compile_definitions(wxbase PRIVATE wxUSE_BASE=1)
-endif()
-
 if(wxUSE_ZLIB)
-    wx_lib_include_directories(wxbase PRIVATE ${ZLIB_INCLUDE_DIRS})
+    wx_lib_include_directories(wxbase ${ZLIB_INCLUDE_DIRS})
```

Comment:
In my mind, I think we should pass `PUBLIC` / `PRIVATE` / `INTERFACE` into `wx_lib_include_directories` so we can easily figure out which keyword we used here.

Reply:
The point of this change is that all the headers that `wxWidgets` includes are `PRIVATE`. It doesn't have to be in the interface, because other projects don't need them. They only need the `wxWidgets` headers.